src/components/Editor/Tabs.js

```diff
@@ -30,6 +30,10 @@ require("./Tabs.css");
  * @returns Immutable.list
  */
 function getHiddenTabs(sourceTabs, sourceTabEls) {
+  if (!sourceTabs) {
+    return;
```

Comment:
I think the initial state for `hiddenTabs` is `null` so it might be best to return that here instead of undefined.  Not sure what's best.